global/prometheus/interconnect.alerts

```diff
@@ -1,12 +1,29 @@
-  ALERT InterconnectDown
+  ALERT RegionInterconnectDown
     IF count((count(probe_success != 1) by (region,region_probed_from) unless (count(probe_success == 1) by (region,region_probed_from) >= 1))) by (region)
     FOR 5m
     LABELS {
+      meta = "{{ $value }} region(s) can’t reach the region",
+      context = "interconnect",
       service = "interconnect",
-      severity = "notyetset",
+      severity = "warning",
```

Comment:
Is `warning` intended? Current interconnect alerts are `critical`. 
Also we would get alerts based on probers and blackbox tests if we merge this. Please turn of the blackbox tests to avoid duplicate alerts.

Reply:
This warning is intended.
The old blackbox test alerts will be turned off on Monday when we saw that these here worked and the dashboard is ready.

Reply:
Ok. Hopefully there will not be too much spam in the meantime.